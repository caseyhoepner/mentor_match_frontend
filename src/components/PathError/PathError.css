.pe-container {
  width: 90%;
  margin: 40px;
}