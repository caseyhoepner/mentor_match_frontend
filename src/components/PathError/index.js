import React from 'react';
import { Link } from 'react-router-dom';

import './PathError.css';

const PathError = () => {

  return (
    <div className='pe-container'>
      <h1>Oops! This page does not exist.</h1>
      <Link 
        className='pe-back-btn' 
        to="/">
        &#8592; Reroute to homepage
      </Link>
    </div>
  )
}

export default PathError;