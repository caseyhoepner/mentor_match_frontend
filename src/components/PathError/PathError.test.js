import React from 'react';
import { shallow } from 'enzyme';
import PathError from '../PathError';

describe ('PathError', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<PathError />)
  })

  it('should match the snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
})