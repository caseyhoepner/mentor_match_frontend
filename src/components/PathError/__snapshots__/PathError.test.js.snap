// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`PathError should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <PathError />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": Array [
        <h1>
          Oops! This page does not exist.
        </h1>,
        <Link
          className="pe-back-btn"
          replace={false}
          to="/"
        >
          ← Reroute to homepage
        </Link>,
      ],
      "className": "pe-container",
    },
    "ref": null,
    "rendered": Array [
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Oops! This page does not exist.",
        },
        "ref": null,
        "rendered": "Oops! This page does not exist.",
        "type": "h1",
      },
      Object {
        "instance": null,
        "key": undefined,
        "nodeType": "class",
        "props": Object {
          "children": "← Reroute to homepage",
          "className": "pe-back-btn",
          "replace": false,
          "to": "/",
        },
        "ref": null,
        "rendered": "← Reroute to homepage",
        "type": [Function],
      },
    ],
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": Array [
          <h1>
            Oops! This page does not exist.
          </h1>,
          <Link
            className="pe-back-btn"
            replace={false}
            to="/"
          >
            ← Reroute to homepage
          </Link>,
        ],
        "className": "pe-container",
      },
      "ref": null,
      "rendered": Array [
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "host",
          "props": Object {
            "children": "Oops! This page does not exist.",
          },
          "ref": null,
          "rendered": "Oops! This page does not exist.",
          "type": "h1",
        },
        Object {
          "instance": null,
          "key": undefined,
          "nodeType": "class",
          "props": Object {
            "children": "← Reroute to homepage",
            "className": "pe-back-btn",
            "replace": false,
            "to": "/",
          },
          "ref": null,
          "rendered": "← Reroute to homepage",
          "type": [Function],
        },
      ],
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
